styles/components/modal.css

```diff
@@ -0,0 +1,20 @@
+#modal, .modal{
+    text-align:center;
+    width:25rem;
+    background-color: var(--clr-blue);
+    padding:1rem
+}
+
+#modal{
+    position: fixed;
+    top:50%;
+    left:50%;
+    transform: translate(-50%, -50%);
+    display:none;
+}
+
+#modal-cancel, .modal-cancel{
+    border:none;
+    background-color:transparent;
+    align-self:flex-end;
+}
```

Comment:
Don't use id selectors as they might cause problems. I am attaching a blog you can refer it. 

https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4#:~:text=IDs%20have%20a%20much%20higher,specificity%20wars%20in%20your%20stylesheets.

Reply:
> Don't use id selectors as they might cause problems. I am attaching a blog you can refer it for better understanding.
> 
> https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4#:~:text=IDs%20have%20a%20much%20higher,specificity%20wars%20in%20your%20stylesheets.

Thanks for mentioning this. Have made the required changes. The article was helpful 😄